sync/src/relayer/tests/compact_block_process.rs

```diff
@@ -145,23 +144,23 @@ fn test_unknow_parent() {
 }
 
 #[test]
-fn test_not_a_better_block() {
+fn test_accept_not_a_better_block() {
     let (relayer, _) = build_chain(5);
-    let chain_state = relayer.shared.lock_chain_state();
-    let header = chain_state.tip_header();
+    let header = {
+        let chain_state = relayer.shared.lock_chain_state();
+        chain_state.tip_header().clone()
+    };
+
+    // The timestamp is random, so it may be not a better block.
```

Comment:
Can we give it less difficulty like [before](https://github.com/nervosnetwork/ckb/pull/1268/files#diff-0ae539c65d2a643736f1fdeb17c7cdb5L158) ?

Reply:
The less difficulty is an invalid header.

https://github.com/nervosnetwork/ckb/blob/develop/sync/src/relayer/compact_block_process.rs#L170

https://github.com/nervosnetwork/ckb/blob/develop/verification/src/header_verifier.rs#L149

Here it's a fork test(valid header but not a better block), not an invalid header test.